src/worker/exporters/node_exporter.py

```diff
@@ -69,43 +74,48 @@ def init_gauges(self):
                     'node_{}'.format(field_name),
                     ['job_id', 'cpu_id', 'numa_domain']
                 )
+            elif 'xid' in field_name:
+                self.gauges[field_name] = prometheus_client.Gauge(
+                    'node_{}'.format(field_name),
+                    'node_{}'.format(field_name),
+                    ['job_id', 'pci_id', 'gpu_id', 'time_stamp']
+                )
+            elif 'link_flap' in field_name:
+                self.gauges[field_name] = prometheus_client.Gauge(
+                    'node_{}'.format(field_name),
+                    'node_{}'.format(field_name),
+                    ['job_id', 'ib_port', 'time_stamp'])
             else:
                 self.gauges[field_name] = prometheus_client.Gauge(
                     'node_{}'.format(field_name),
                     'node_{}'.format(field_name),
                     ['job_id']
                 )
```

Comment:
Assuming that only labels are going to change for each new gauge/metric the gauge could be instantiated outside of the if